### PR TITLE
improved API interface

### PR DESCRIPTION
**Pull request by ninja-inc**
I improved API interface.

### CODE REVIEW

File: test/response/api/v1/attendances/get.json [hunk 1]
```diff
@@ -1,25 +1,33 @@
...
+      },
+      "attendance": {
+        "id": 1,
+        "href": "/attendances/1"
+      },
+      "remarks": null,
+      "createdAt": "2016-11-18T14:29:05.000Z",
+      "updatedAt": null,
+      "attendanceAt": "2016-12-26",
+      "start": null,
```
Comment (sideroad) [inline test/response/api/v1/attendances/get.json]: startTime

File: test/response/api/v1/attendances/get.json [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
+      "end": null,
```
Comment (sideroad) [inline test/response/api/v1/attendances/get.json]: endTime

File: test/response/api/v1/attendances/get.json [hunk 3]
```diff
@@ -1,25 +1,33 @@
 {
   "attengonces": [
```
Comment (sideroad) [inline test/response/api/v1/attendances/get.json]: items

File: test/response/api/v1/attendances/get.json [hunk 4]
```diff
@@ -1,25 +1,33 @@
...
+      },
+      "attendanceType": {
+        "id": 5,
+        "href": "/attendanceTypes/5"
+      },
+      "reason": {
+        "id": 6,
+        "href": "/reasons/6"
+      },
+      "attendance": {
```
Comment (sideroad) [inline test/response/api/v1/attendances/get.json]: remove

File: test/response/api/v1/attendances/get.json [hunk 5]
```diff
@@ -1,25 +1,33 @@
...
+      },
+      "member": {
+        "id": 1,
+        "href": "members/1"
+      },
+      "attendanceType": {
+        "id": 5,
+        "href": "/attendanceTypes/5"
+      },
+      "reason": {
```
Comment (sideroad) [inline test/response/api/v1/attendances/get.json]: attendanceReason

File: test/response/api/v1/attendances/get.json [hunk 6]
```diff
@@ -1,25 +1,33 @@
...
+        "id": 1,
+        "href": "/attendances/1"
+      },
+      "remarks": null,
+      "createdAt": "2016-11-18T14:29:05.000Z",
+      "updatedAt": null,
+      "attendanceAt": "2016-12-26",
+      "start": null,
+      "end": null,
       "start_to_end": ""
```
Comment (sideroad) [inline test/response/api/v1/attendances/get.json]: Remove

File: test/response/api/v1/attendances/get.json (hunk 3 above)
Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json]: modified

File: test/response/api/v1/attendances/get.json (hunk 5 above)
Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json]: modified

File: test/response/api/v1/attendances/get.json (hunk 4 above)
Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json]: removed

File: test/response/api/v1/attendances/get.json (hunk 1 above)
Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json]: changed

File: test/response/api/v1/attendances/get.json (hunk 2 above)
Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json]: changed

File: test/response/api/v1/attendances/get.json (hunk 6 above)
Comment (ninja-inc) [reply test/response/api/v1/attendances/get.json]: removed

**Review by da-shino — APPROVED** [review]

**Pull request merged**